contracts/router/AedexV2Router.aes

```diff
@@ -36,7 +36,7 @@ payable contract AedexV2Router =
 
     // **** ADD LIQUIDITY ****
 
-    //just creates the pair if it is the case and calculates the amount
+    // just creates the pair if it is the case and calculates the amount
```

Comment:
I find it difficult to understand this line. What does _if it is the case_ mean here?

Reply:
It is addressed and modified in this PR #59